src/resources/postcss/components/_small-cta.pcss

```diff
@@ -0,0 +1,31 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Small CTA (Tooltips - ET, ET+, RSVP)
+ *
+ * ----------------------------------------------------------------------------- */
+
+.tribe-events {
+
+	.tribe-events-c-small-cta-link {
+		@mixin desktop-body-3-bold;
+
+		border-bottom: 2px solid var(--color-border-default);
```

Comment:
I think that this is supposed to be `--color-accent-primary`, and the hover state is only the text color changing? Maybe check with Matt.

Reply:
actually @ryanurban is right here, we should provide a variation of the existing CTA from common instead of redefining the CTA styles again.

Reply:
The CTA we have in common is pretty limited. https://moderntribe.invisionapp.com/d/main#/console/17163687/359338308/preview

Do you mean moving this to common?

Reply:
Yeah, you know what, I think it might be a good idea to move over to common as an `--alt` case.

Reply:
check out https://github.com/moderntribe/tribe-common/pull/1029, i've added the alt case in common while doing a kitchen sink update.

Reply:
Ok, adapted this to that code. The component in common is still missing compat with themes. Please review the changes.